src/base/js/honeycomb.base.js

```diff
@@ -0,0 +1,4 @@
+var Honeycomb = Honeycomb || {};
+
+Honeycomb.version = '1.0.2';
```

Comment:
I might rename this to semanticVersion to make it clear you're using semantic versioning